extensions/dashlane-vault/Sources/auth/main.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import LocalAuthentication
+
+let policy = LAPolicy.deviceOwnerAuthenticationWithBiometrics
```

Comment:
This policy here is insufficient, it can be bypassed, you should rather use `kSecAttrAccessControl`.

I recommend you to have a read to this post: https://www.andyibanez.com/posts/ios-keychain-touch-id-face-id/

Reply:
I looked into this but couldn't get it working as `kSecAttrAccessControl` requires "Keychain Access Groups" entitlement which is not supported by CLI Apps and also requires code signing. Workaround seems to be using a separate daemon for this.

Given the scope it would make more for Dashlane to integrate this natively in a secure manner, I expect this to take quite some time assuming your team approves this feature.

However in the meantime we could still merge this since something is better than simply leaving the vault open at all times which is a huge security hazard.